client/src/components/RoomsList.tsx

```diff
@@ -41,13 +41,13 @@ const RoomsList: React.FC = () => {
 
       {loading && <Loading />}
       {error && <span>Error Loading Rooms</span>}
-      {data?.currentUserRooms?.map((room, index) => {
+      {data?.currentUserRooms?.map(room => {
         return (
           <RoomLink
             id={room.id}
             key={room.id}
             name={room.name}
-            isSelected={false}
+            isSelected={!!history.location.pathname.match(room.id)}
```

Comment:
Since the <RoomsList /> component is outside the <Route /> component i don't have access to useParam hook to get the current `roomId` that's why i string matched the `roomId`